src/components/Metrics/index.js

```diff
@@ -9,6 +9,10 @@ import useStyles from "./useStyles";
 import Card from "@/pesayetu/components/Card";
 import Section from "@/pesayetu/components/Section";
 
+// NOTE(nyokabi) useStyles uses import/definition order to determine how
+//                 classes are ordered.
+//                 see: https://material-ui.com/styles/advanced/#makestyles-withstyles-styled
+
```

Comment:
What's up with this comment?

Reply:
@kilemensi This was to fix the reload issue for images. The styles should come after this. Let me fix that.

Reply:
@kilemensi Um, I think it should be good now. Added the comment /rule to support why usestyles has low precedence on imports. Otherwise every time we reload the page the card styles added on media disappear